### DISCUSSION TITLE
Can this emulate multiple controllers in multiplayer games?

### DISCUSSION THREAD

**Discussion by 2ndHouse** [r57zone/DualShock4-emulator#64] [Q&A] [answered]
Looked around, but couldn't find the answer. Thought I would ask here.  DS4 emulator has worked pretty well on the single player games I've tried, but just tried it for the first time in a local co-op game and having a problem.

With the emulator running, the second controller does not get recognized at all in game. Disabling the emulator and restarting PS Plus makes the second controller detectable again, but losing touchpad emulation isn't great. The game is Alienation, and using two wired Xbox Series S/X controllers.

**Comment by r57zone** [comment, marked as answer]
Currently DS4Emulator does not support more than one controller, but Discord has come up with a way to make it possible to make more than one controller.

1. connect 1st controller,
2. start DS4Emulator
3. Hide 1st xbox controller in HideHide (leave the sony emulated one enabled)
4. connect 2nd controller,
5. start a second instance of DS4Emulator
6. Hide 2nd xbox controller in HideHide
7. play 🙂 and you'll have just 2 sony controllers in any game

**Reply by 2ndHouse** [reply]
Thanks for the help! Will have to try this out when I get the chance.
